src/content/docs/journey-demand/find-your-baseline.mdx

```diff
@@ -0,0 +1,185 @@
+
+    </Step>
+    <Step>
+    
+    ### Find latency baseline and success baseline
+
+    Focusing on `/js/controllers/`, update the above query to:
+
+    * Remove the `FACET` and focus the query on that particular URI
+    * Replace the total `count(*)` with `percentile(duration, 95)`
+
+<SideBySide>
+    <Side>
+
+        ```sql
+        FROM Transaction SELECT percentile(duration, 95) AS 'Latency Baselines', percentage(count(*), WHERE error is false) AS 'Success Baseline' SINCE 1 WEEK AGO WHERE request.URI LIKE '/js/services/%'
+        ```
+    </Side>
+    <Side>
+
+<img
+    title="Latency and Success baseline"
+    alt=""
+    src={queriesnrqlLatencyandSuccessBaseline}
+/>
+    </Side>
+</SideBySide>
+
+This query tells us that the transaction typically responds at 42(ms) and has a 99.27% success rate. This is our latency baseline. 
+
+</Step>
+<Step>
+
+### Create baseline-informed service levels
+
+Now that you have some baselines, you can create a meaningful service level. From the **Query Your Data** page, head back to **[one.newrelic.com](https://one.newrelic.com) > APM & Services**, then click **Service levels** located under **Reports**.
+
+When you add a new service level, New Relic will auto-populate a baseline for every data source in your entire app. While useful in some cases, we're wanting to monitor specific capabilities that could have catastrophic consequences on a peak demand day. 
+
+With the baseline we pulled from the previous section, we can edit the `WHERE` box. Add the following string to the end of the populated query so the line reads:
+
+<SideBySide>
+
+    <Side>
+```sql
+entityGUID = 'YOUR_GUID' AND (transactionType = 'Web') AND request.uri LIKE `/js/services/%`
+```    
+    </Side>
+    <Side>
+<img
+    title="Create baseline informed SLI"
+    alt=""
+    src={slmUpdateSLIinUI}
+/>    
+</Side>
+</SideBySide>
+
+After you've updated the `WHERE` field, confirm that your duration in the `AND` field matches the time in miliseconds queried in step 2. In this case, the request gets a response in about 42(ms) and the `AND` field matches with a .4 duration.
+
+</Step>
+<Step>
+    ### View your service levels in the UI
+
+Now that you've created a baseline-informed service level, you can now view it in the UI. We recommend that you create an alert for this service level, which you can evaluate the quality of in the next part of the tutorial series.
+
+<SideBySide>
+
+    <Side>
+
+    <img
+    title="View your service levels"
+    alt=""
+    src={serviceLevelsintheUIServiceLevelsintheUI}
+/>
+    </Side>
+
+    <Side>
+        <img
+        title="Service levels summary page"
+        src={serviceLevelsUI}
+        />
+    </Side>
+
+</SideBySide>
+
+</Step>
+</Steps>
```

Comment:
confirm with andrew if we can add script info here / confirm with austin if that has a place here